priam/src/main/java/com/netflix/priam/identity/token/TokenRetrieverUtils.java

```diff
@@ -155,4 +156,22 @@ public GossipParseException(String message, Throwable t) {
             super(message, t);
         }
     }
+
+    /** This exception is thrown either when a node is bootstrapping using a token that is alive. */
+    public static class TokenAliveException extends Exception {
+
+        private static final long serialVersionUID = 1038678311186020257L;
+
+        public TokenAliveException() {
```

Comment:
Looks like codecov is complaining because these overloads are not tested. They aren't called so I recommend to just remove them.

Specifically:

public TokenAliveException()
public TokenAliveException(String message, Throwable t)